indexer/packages/postgres/src/stores/affiliate-info-table.ts

```diff
@@ -94,6 +97,186 @@ export async function findById(
     .returning('*');
 }
 
+/**
+ * Updates affiliate information in the database based on the provided time window.
+ *
+ * This function aggregates affiliate-related metadata and fill statistics
+ * from various tables. Then it upserts the aggregated data into the `affiliate_info` table.
+ *
+ * @async
+ * @function updateInfo
+ * @param {string} windowStartTs - The exclusive start timestamp for filtering fills.
+ * @param {string} windowEndTs - The inclusive end timestamp for filtering fill.
+ * @param {Options} [options={ txId: undefined }] - Optional transaction ID or additional options.
+ * @returns {Promise<void>}
+ */
+export async function updateInfo(
+  windowStartTs: string, // exclusive
+  windowEndTs: string, // inclusive
+  options: Options = { txId: undefined },
```

Comment:
I think you should just accept txId directly here instead of using options. Options have a [readReplica](https://github.com/dydxprotocol/v4-chain/blob/main/indexer/packages/postgres/src/types/utility-types.ts#L20) field that if populated, one would assume you would use the readReplica. However for this type of operation, we would never accept readReplica = true, because it is a write operation